CONTRIBUTING.md

```diff
@@ -53,8 +53,9 @@ Atomic commits will make it easier to track down regressions. Also, it enables t
 1. Fork it (https://github.com/AndroidDev-social/MastodonCompose/fork)
 2. Create a new feature branch (`git checkout -b feature/fooBar`)
 3. Commit your changes (`git commit -am 'Add some fooBar'`)
-4. Push to the branch (`git push origin feature/fooBar`)
-5. Create a new Pull Request
+4. Run `./gradlew detekt` and fix any code issues before pushing
```

Comment:
I would run `./gradlew check` to run all other tests and checks as well.

Reply:
Good idea. Though Detekt hasn't been enabled to work with the check task yet (until everything has been properly formatted we can enable it)

Reply:
Okay, then we can keep it for now and change it later when it is fixed and enabled.

Reply:
Should be changed on the next PR that fixes all the detekt issues